DependencyInjection/Configuration.php

```diff
@@ -17,8 +17,15 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('fervo_enum');
+        $treeBuilder = new TreeBuilder('fervo_enum');
+
+        if (method_exists($treeBuilder, 'getRootNode')) {
+            $rootNode = $treeBuilder->getRootNode();
+        } else {
+            // BC layer for symfony/config 4.1 and older
+            $rootNode = $treeBuilder->root('fervo_enum');
+        }
+
```

Comment:
this extra blank line is not necessary

Reply:
Force-pushed the new code with the blank line removed